.github/workflows/go-tests.yaml

```diff
@@ -50,11 +50,11 @@ jobs:
 
       - name: Prepare samples
         run: |
-          cp -a ${{ github.workspace }}/test_data/. ${{ github.workspace }}/samples/
```

Comment:
Can we run the makefile rule instead of duplicating this logic?

Reply:
I had some hang up about that mentally but it may just be because I cloned the other repository separately and wasn't sure if our anonymous clone would work.

The make targets should work in CI as well.

Reply:
Updated in [`4ad123f` (#448)](https://github.com/chainguard-dev/bincapz/pull/448/commits/4ad123f821ebef10b812869682c6d1b38610c9a0).